components/zcash_address/src/kind/unified/fvk.rs

```diff
@@ -200,7 +208,9 @@ mod tests {
             shielded in arb_shielded_fvk(),
             transparent in prop::option::of(arb_transparent_fvk()),
         ) -> Ufvk {
-            Ufvk(shielded.into_iter().chain(transparent).collect())
+            let mut items = shielded.into_iter().chain(transparent).collect::<Vec<_>>();
```

Comment:
Style nit:
```suggestion
            let mut items: Vec<_> = shielded.into_iter().chain(transparent).collect();
```
(The turbofish is fine when needed, but I prefer to put the type up-front when I can.)